lib/experimental/new_embed.dart

```diff
@@ -614,6 +615,109 @@ class FlashBox {
   void hide() {
     _element.setAttr('hidden');
   }
+
+  void show() {
+    _element.clearAttr('hidden');
+  }
+}
+
+class AnalysisResultsElement extends DElement {
+  static const String _noIssuesMsg = 'no issues';
+  static const String _hideMsg = 'hide';
+  static const String _showMsg = 'show';
+
+  DElement message;
+  DElement toggle;
+  bool _hidden = true;
+
+  final StreamController<AnalysisIssue> _onClickSink =
```

Comment:
nit: Would `_onClickController` be more appropriate here, since it's not actually typed as `Sink<AnalysisIssue>`?